src/components/Modal/node/SendMessageModal.tsx

```diff
@@ -117,25 +117,35 @@ export const SendMessageModal = (props: SendMessageModalProps) => {
       }
     }
     if (sendMode == 'path') {
-      const pathElements = path.replace(/(\r\n|\n|\r| )/gm, '').split(',');
+      const pathElements = path
+        .split('\n')
+        .map((line) =>
+          line
+            .split(',')
+            .map((element) => element.trim())
+            .filter((element) => element !== '')
+        )
+        .flat();
```

Comment:
What are the reasons for replacing .replace with map in a map and then flat? Is that more performant?

Reply:
The change is not related to performance, more about being able to easlly understand what is happening.
With this change we can follow the though process, (after the map line) about splitting the commas, removing the whitespace, if the element is an empty string, then we don't need it since it is a product of a consecutive comma.

We could do it in regex if it is preffered in this case.
the regex would be something like `.replace(/,+/g, ',').split(',');`

Reply:
Unfetunetely .map inside a map, and split after slit is not easy to read.
Please make it easier to read. If you need help, let me know